bot/pagination.py

```diff
@@ -327,50 +297,30 @@ def add_line(self, line: str = '', *, empty: bool = False) -> None:
         self.close_page()
 
     def add_image(self, image: str = None) -> None:
-        """
-        Adds an image to a page
-        :param image: image url to be appended
-        """
-
+        """Adds an image to a page."""
         self.images.append(image)
 
     @classmethod
     async def paginate(cls, pages: List[Tuple[str, str]], ctx: Context, embed: Embed,
                        prefix: str = "", suffix: str = "", timeout: int = 300,
                        exception_on_empty_embed: bool = False):
```

Comment:
This `classmethod` does not have a return annotation. https://github.com/python-discord/bot/blob/138567f371fa2db8675c2b5661ff5526e774335a/bot/pagination.py#L303-L306

There's at least one path that has the `return await ctx.send` pattern, so it's an `Optioonal[Message]` (`Message` should still be added to the import list, see previous comment.)
```suggestion
                       exception_on_empty_embed: bool = False) -> Optional[Message]:
```